src/vs/workbench/contrib/chat/browser/chatSessions/view/sessionsTreeRenderer.ts

```diff
@@ -305,17 +253,29 @@ export class SessionsRenderer extends Disposable implements ITreeRenderer<IChatS
 				} : undefined) :
 			undefined;
 
-		// Set the resource label
-		templateData.resourceLabel.setResource({
-			name: session.label,
-			description: !renderDescriptionOnSecondRow && 'description' in session && typeof session.description === 'string' ? session.description : '',
-			resource: iconResource
-		}, {
-			fileKind: undefined,
-			icon: iconTheme,
-			// Set tooltip on resourceLabel only for single-row items
-			title: !renderDescriptionOnSecondRow || !session.description ? tooltipContent : undefined
-		});
+		// Handle custom icon rendering
+		if (iconTheme) {
+			templateData.customIcon.className = `chat-session-custom-icon ${ThemeIcon.asClassName(iconTheme)}`;
+			if (iconTheme.color) {
```

Comment:
We don't support color anymore, we could remove this

Reply:
Removed